lang/cs/Org.Apache.REEF.IMRU.Tests/MapperCountTest.cs

```diff
@@ -46,7 +46,7 @@ public void TestMapperCountInProcess()
                             .Build()
                     )
                     .GetInstance<MapperCount>();
-            var result = tested.Run(NumberOfMappers);
+            var result = tested.Run(NumberOfMappers, "", TangFactory.GetTang().NewConfigurationBuilder().Build());
```

Comment:
Once you remove FileSysmConfiguration from Run method, this dummy configuration would not needed.